### ISSUE TITLE
Make all pages and features fully accessible to screen reader software

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-web#267]
Test with at least VoiceOver.

**Comment by mitchdowney** [comment]
The website should be mostly accessible by now. Any accessibility bugs should be created as separate issues.
